### ISSUE TITLE
Exclude folder not working

### ISSUE THREAD

**Issue by codetestmg** [laravel/pint#307]
### Pint Version

1.16

### PHP Version

8.2.4

### Description

I have following pint.json

```
{
    "preset": "laravel",
    "exclude": [
        "bootstrap/cache",
       "bootstrap/cache/*",
        "bootstrap/cache/*.php",

    ]
}
```

but when i ran following commnad

`./vendor/bin/pint bootstrap`

It scans cache folder

### Steps To Reproduce

I have following pint.json

```
{
    "preset": "laravel",
    "exclude": [
        "bootstrap/cache",
       "bootstrap/cache/*",
        "bootstrap/cache/*.php",

    ]
}
```

but when i ran following commnad

`./vendor/bin/pint bootstrap`

It scans cache folder

**Comment by crynobone** [comment]
This seems to be the expected behavior by `php-cs-fixer`: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/2956#issuecomment-319677923

**Comment by joelbladt** [comment]
Hello @codetestmg ,

Your `pint.json` is invalid. In `exclude` the last element item dosen't need a comma at the end of line. Never. And the next thing: I don't understand why you are using the preset laravel and describe the exclude for the bootstrap files.

Why this question? If no `pint.json` will be found in your Project or a defined preset, the laravel preset will be used per default. See `app/Repositories/ConfigurationJsonRepository.php` at line 69

And in the Factories the `bootstrap/cache` folder will be automatically exluded. See `app/Factories/ConfigurationFactory.php` at line 30

**Comment by codetestmg** [comment]
@joelbladt  Thanks .But

Even if i remove comma and preset also output is same 


```
{

    "exclude": [
        "bootstrap/cache",
        "bootstrap/cache/*",
        "bootstrap/cache/**"
    ]
}
```



Output : 


  ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────── Laravel

    FIXED   ............................................................................................................................................................................................ 4 files, 3 style issues fixed

───────────────────────────────────────────────────────────────── Laravel
    FIXED   ............................................................................................................................................................................................ 4 files, 3 style issues fixed

................................................................................................ 4 files, 3 style issues fixed
  ✓ bootstrap\cache\events.php                                     array_syntax, single_space_around_construct, no_multiline_whitespace_around_double_arrow, array_indentation, blank_line_after_opening_tag, single_blank_line_at_eof


  ✓ bootstrap\cache\packages.php                                   array_syntax, single_space_around_construct, no_multiline_whitespace_around_double_arrow, array_indentation, blank_line_after_opening_tag, single_blank_line_at_eof


  ✓ bootstrap\cache\services.php                                   array_syntax, single_space_around_construct, no_multiline_whitespace_around_double_arrow, array_indentation, blank_line_after_opening_tag, single_blank_line_at_eof

**Comment by tom-mfry** [comment]
I came across the exact same issue and found the solution: https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/issues/2956#issuecomment-319713451

When pint is run with a path argument, all excludes inside the `pint.json` file are ignored. Instead, if you just run `vendor/bin/pint`, you should find that your `bootstrap/cache` file is ignored. I had the same issue.

**Comment by codetestmg** [comment]
> Hey there,
> 
> While this may be a legitimate issue, can you first try posting your problem or question on one of the support channels below? If this issue can be definitively identified as a bug, feel free to open up a new issue with a link to the original one and we'll gladly help you out.
> 
> * [Laracasts Forums](https://laracasts.com/discuss)
> * [Laravel.io Forums](https://laravel.io/forum)
> * [StackOverflow](https://stackoverflow.com/questions/tagged/laravel)
> * [Discord](https://discordapp.com/invite/KxwQuKb)
> * [Larachat](https://larachat.co/)
> * [IRC](https://web.libera.chat/?nick=laravelnewbie&channels=#laravel)
> 
> Thanks!

@crynobone  .please check @tom-mfry  comment

**Comment by crynobone** [comment]
Yes and it inline with what is expected since he suggest running `vendor/bin/pint` without any argument.

**Comment by Tamas-hi** [comment]
I don't understand how this can be the expected behaviour though from PHP-CS-Fixer.
If you are using lint-staged and wants to format only a single changed file, config is ignored?

Is someone has a recommandation how to do this the right way, I would thank that.
